Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.7.7-slim
+
+# Create virtualenv and add to path.
+ENV VIRTUAL_ENV=/opt/venv
+RUN python3 -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
```

Comment:
Is it a good practice to use a virtualenv within a Docker container?

Reply:
Don't see the point - everything is root anyway.

Reply:
Some light googling suggests there is a point when doing multi-stage builds as it's easier to copy over just the site-packages that your application uses: https://pmac.io/2019/02/multi-stage-dockerfile-and-python-virtualenv/

Reply:
That is true, but you aren't copying the files between stages in this Dockerfile :-P I plan on moving the kraken-core image to this at some point, once I work out how to copy from a venv to the global python modules.